src/core/adaptors/organizations/index.adaptors.ts

```diff
@@ -40,7 +40,7 @@ export const getOrgIdAdaptor = async (): Promise<AdaptorRes<string>> => {
   try {
     const { results = [] } = await getOrgs();
     return {
-      data: results[0]?.id || '',
+      data: results && results.length ? results[0]?.id || '' : '',
```

Comment:
No need to extra checking 
Because default value for results is empty array 
- If `getOrgs()` has not results in object, so results will be empty array -> no need to check `results &&`
- if results is an empty array, then results[0] will be `undefined` and accessing to `id` with `?.` (optional) will be `undefined` too so executed code will be `undefined || ' '` (that will be empty string) -> no need to check `results.length &&`

Reply:
No, some users who are registered in the first days , have some issues in data, e.g. my own user, the organization for my user is null and the app crashes when I login with it. I added this extra check to avoid it.

Reply:
I checked before with thousand new accounts and didn't pass organization so organization id will be null and there was no crash in app.
Let me talk you today.